doc/nrf/gs_installing.rst

```diff
@@ -55,6 +55,61 @@ The installation process is different depending on your operating system.
 
       Also see :ref:`zephyr:mac-setup-alts` for additional information.
 
+If you are interested in building `Connected Home over IP`_ applications, install also the `GN`_ meta-build system to generate files for `Ninja`_.
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      The Connected Home over IP project currenlty supports only Linux and macOS.
+      Support for Windows is not yet available.
+
+   .. group-tab:: Linux
+
+      To install the GN tool, complete the following steps:
+
+      1. Download the GN binary archive and extract it by using the following commands:
+
+         .. parsed-literal::
+            :class: highlight
+
+            mkdir ${HOME}/gn && cd ${HOME}/gn
+            wget -O gn.zip https://chrome-infra-packages.appspot.com/dl/gn/gn/linux-amd64/+/latest
+            unzip gn.zip gn
+            rm gn.zip
+
+      #. Add the location of the GN tool to the system PATH.
+         For example, if you are using ``bashrc``, run the following commands in the directory with extracted executable file:
+
+         .. parsed-literal::
+            :class: highlight
+
+            echo 'export PATH=${HOME}/gn:"$PATH"' >> ${HOME}/.bashrc
+            source ${HOME}/.bashrc
+
+   .. group-tab:: macOS
+
+      To install the GN tool, complete the following steps:
```

Comment:
Probably the same changes as for Linux, but I'm not familiar with how it works on macOS.